wptdash/date.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from datetime import date
+import math
+
+
+def get_quarter_start_date(quarter, year):
+    quarter_first_month = quarter * 3 - 2
+    quarter_start = date(year, quarter_first_month, 1)
+    return quarter_start
+
+
+def get_default_start_end():
+    month_end_days = {
+        3: 31,
+        6: 30,
+        9: 30,
+        12: 31
+    }
+    today = date.today()
+    month = today.month
+    year = today.year
+    quarter = math.ceil(month / 3)
+    quarter_start = get_quarter_start_date(quarter, year)
+    delta = today - quarter_start
+    if delta.days < 45:
```

Comment:
A comment explaining the number 45 would be nice.

Reply:
:+1: